MyFirstProject/NikitaSosyukHomework/StatusView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  StatusView.swift
+//  NikitaSosyukHomework
+//
+//  Created by Nikita Sosyuk on 20.09.2020.
+//  Copyright © 2020 Amir Zigangaraev. All rights reserved.
+//
+
+import UIKit
+
+class StatusView: UIView {
+    
+    @IBOutlet private var statusLabel: UILabel!
+    
+    func load() {
+        statusLabel.text = "Need to sign in to continue."
+        statusLabel.textColor = .black
+        statusLabel.textAlignment = .center
+        statusLabel.font = UIFont.systemFont(ofSize: 23.0)
+        self.backgroundColor = .systemGray6
+        self.layer.cornerRadius = 10
+    }
```

Comment:
Если сделать вот так, тебе не придётся нигде эту функцию вызывать, она вызовется автоматически.

```suggestion
    override func awakeFromNib() {
        super.awakeFromNib()
        statusLabel.text = "Need to sign in to continue."
        statusLabel.textColor = .black
        statusLabel.textAlignment = .center
        statusLabel.font = UIFont.systemFont(ofSize: 23.0)
        self.backgroundColor = .systemGray6
        self.layer.cornerRadius = 10
    }
```